src/applause/common_python_reporter/auto_api.py

```diff
@@ -270,10 +270,10 @@ def upload_asset(
         requests.post(
             f"{self.config.auto_api_base_url}api/v1.0/test-result/{result_id}/upload",
             headers=headers,
-            files={file: file},
+            files={"file": (asset_name, file, "application/octet-stream")},
```

Comment:
Is this formatted correctly? Do we need content-type: here or something like that?

Reply:
Yes, the content type is implicitly set by the requests library as "multipart/form-data" and the application/octet-stream is placed correctly for the file content type